store/adapters/rdbms/drivers/mysql/dialect.go

```diff
@@ -25,21 +23,64 @@ var (
 	dialect            = &mysqlDialect{}
 	goquDialectWrapper = goqu.Dialect("mysql")
 	quoteIdent         = string(mysql.DialectOptions().QuoteRune)
+
+	nuances = drivers.Nuances{
+		HavingClauseMustUseAlias: false,
```

Comment:
In the comment above you write `// MySQL requires use of aliases inside HAVING clause` did you mean some other DB?

Reply:
Correct, it is other way around.